.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and push docker image omnisette-server
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
```

Comment:
Why use GitHub's registry over Docker Hub? Doesn't that require logging into GitHub to pull it? Or am I misremembering that...

Reply:
I'm just not sure if end-users will need to authenticate with GitHub to download it. It's required for other GitHub stuff, even public actions artifacts, so I'm just a bit concerned.